.github/CONTRIBUTING.md

```diff
@@ -69,10 +69,12 @@ git rebase master
 git push --set-upstream origin feature/325-japanese-translation
 ```
 
-Finally, go to GitHub and [make a Pull Request][] :D
+Finally, go to GitHub and [make a Pull Request](https://github.com/SAP/fundamental/pulls) :D
 
+In the subject of the pull request, use "feat:" to denote an enhancement, "fix:" to denote a bug fix, "chore:" for small configuration updates or "docs:" for documentation updates and briefly describe the bug fix or enhancement you're contributing.
```

Comment:
Is it worth putting `feat:`, `fix:`, `chore:` and `docs:` in back ticks instead of quotes?